cypress/support/step-definitions/accordion-steps.js

```diff
@@ -0,0 +1,95 @@
+import {
+  accordionTitleContainer, accordionIcon, accordion,
+} from '../../locators/accordion';
+import { eventInAction } from '../../locators';
+
+const FIRST_ELEMENT = 0;
+
+Then('Accordion iconAlign property on preview is set to {string}', (iconAlign) => {
+  accordionTitleContainer().eq(FIRST_ELEMENT).children().first()
+    .should('have.attr', 'data-element', 'accordion-title')
+    .should('be.visible');
+  accordionTitleContainer().eq(FIRST_ELEMENT).children().last()
+    .should('have.attr', 'data-component', 'icon')
+    .should('be.visible');
+  if (iconAlign === 'right') { // set by default
+    accordionTitleContainer().should('have.css', 'justify-content', 'space-between')
+      .and('not.have.css', 'flex-direction', 'row-reverse');
+    accordionTitleContainer().eq(FIRST_ELEMENT).children().first()
+      .should('have.css', 'margin-right', '0px');
+  } else {
+    accordionTitleContainer().should('have.css', 'flex-direction', 'row-reverse');
+    accordionTitleContainer().eq(FIRST_ELEMENT).children().last()
+      .should('have.css', 'margin-right', '16px');
+  }
+});
+
+Then('Accordion iconType property on preview is set to {string}', (iconType) => {
+  accordionIcon().should('have.attr', 'data-element', iconType).should('be.visible');
+});
+
+Then('Accordion type property on preview is set to {string}', (type) => {
+  if (type === 'primary') {
+    accordion().should('have.css', 'background-color', 'rgb(255, 255, 255)');
+    accordion().should('have.css', 'border', '1px solid rgb(204, 214, 218)');
+  } else {
+    accordion().should('have.css', 'background-color', 'rgba(0, 0, 0, 0)');
+    accordion().should('not.have.css', 'border', '1px solid rgb(204, 214, 218)');
+  }
+});
+
+When('I expand accordionRow via click', () => {
+  accordionTitleContainer().click();
+});
+
+When('I expand accordionRow using enter key', () => {
+  accordionTitleContainer().trigger('keydown', { keyCode: 13, which: 13 });
+});
+
+Then('accordionRow is expanded', () => {
+  accordionTitleContainer().should('have.attr', 'aria-expanded', 'true').and('be.visible');
+});
+
+Then('Accordion has proper primary type color palette', () => {
```

Comment:
use parameter `color` for `background-color`

Reply:
updated